### PR TITLE
Maintenance/fix warnings

### PR DESCRIPTION
**Pull request by ryan-kipawa**
A bit of maintenance with new versions of GeoPandas and Pandas being released

### CODE REVIEW

**Comment by ecomodeller** [comment]
FYI Python 3.8 is supported by latest version of pandas https://pypi.org/project/pandas/#files

File: tests/test_result_frame_aggregator.py [hunk 1]
```diff
@@ -345,10 +345,10 @@ def test_max_aggregation(self, res1d_river_network, column_mode):
         df_agg = df_agg.groupby("name").agg("max")  # test for entire reach (not segments)
 
         assert list(df_agg.columns.values) == [
+            "max_WaterLevel",
```
Comment (gedaskir) [inline tests/test_result_frame_aggregator.py:348]: What is the reason to shuffle this around?

**Comment by gedaskir** [comment]
> FYI Python 3.8 is **not** supported by latest version of pandas https://pypi.org/project/pandas/#files

Should we remove Python 3.8 support? It is reaching end-of-life in 2024 Q4,

**Comment by ecomodeller** [comment]
> > FYI Python 3.8 is **not** supported by latest version of pandas https://pypi.org/project/pandas/#files
> 
> Should we remove Python 3.8 support? It is reaching end-of-life in 2024 Q4,

I think so, yes.

**Comment by gedaskir** [comment]
I saw that the output of some notebooks is ditched. I personally like it to be there when statically viewing it online. I suggest to revert this change.

File: tests/test_result_frame_aggregator.py (hunk 1 above)
Comment (ryan-kipawa) [reply tests/test_result_frame_aggregator.py:348]: The new stack algorithm used in future pandas has a different order than before unfortunately

**Comment by ryan-kipawa** [comment]
> I saw that the output of some notebooks is ditched. I personally like it to be there when statically viewing it online. I suggest to revert this change.

Yes I agree with this. My thinking was mainly to avoid messy merge conflicts with upcoming branches

**Review by gedaskir — APPROVED** [review]
Looks good!

**Pull request merged**